TAScheduler/UnitTests/tests.py

```diff
@@ -23,12 +27,14 @@ def setUp(self):
         self.adminObj = AdminObj(admin_info)
 
     def test_login_valid_credentials(self):
-        result = self.adminObj.login("test@example.com", "password123")
-        self.assertRedirects(result, '/home/', msg="login with vaild credentials failed to redirect to home")
+        print("Testing with email: admin@example.com, password: adminpass")
+        result = self.adminObj.login("admin@example.com", "adminpass")
+        self.assertTrue(result, "login with valid credentials failed")
 
     def test_login_invalid_credentials(self):
-        result = self.userObj.login("test@example.com", "wrongpassword")
-        self.assertRedirects(result, '/', msg="incorrect password failed to redirect to login")
+        # Assuming the login method returns False for invalid credentials
+        result = self.adminObj.login("admin@example.com", "wrongpassword")
+        self.assertFalse(result, "login with invalid credentials should fail but didn't")
```

Comment:
both of those should already return false, because with the wrong username, there won't be a user to compare passwords, so it wont login, and empty will also not find any user, and won't log in.

Reply:
I agree - they "should" return false, but our UnitTests are to ensure that is actually happening. I think I am misunderstanding you: if you are saying you're already testing these scenarios, could you quickly point out the line numbers but I currently do not see how these are already tested.

I still think you need to add a few quick tests to ensure nothing weird happens - the login is one of the most important functions because it sets up the entire program imo.

Reply:
Good point, Ill add them.